sbol_utilities/package.py

```diff
@@ -1,13 +1,13 @@
 import os
-import glob
-import warnings
+# import glob
```

Comment:
If something isn't needed, please delete instead of commenting out.

Reply:
Resolved in commit "remove unneeded packages"